src/utils/venue.ts

```diff
@@ -28,8 +28,11 @@ export const canBeDeleted = (venue: AnyVenue): boolean =>
 export const canHavePlacement = (venue: AnyVenue): boolean =>
   PLAYA_TEMPLATES.includes(venue.template);
 
-export const checkIfValidVenueId = (venueId: string): boolean =>
-  /[a-z0-9_]{1,250}/.test(venueId);
+export const checkIfValidVenueId = (venueId?: string): boolean => {
+  if (typeof venueId !== "string") return false;
+
+  return /[a-z0-9_]{1,250}/.test(venueId);
+};
```

Comment:
[note] This change is mostly to keep this implementation aligned with the backend version, but essentially, `.test` can only take a `string`, and if anything else is passed to it (eg. `undefined`, `null`, etc), then it will seemingly implicitly call `.toString()` on it, which often then causes the value to pass our test here even though it shouldn't (eg. because the string `"undefined"` matches the regex)